CMakeLists.txt

```diff
@@ -23,10 +23,14 @@ set(CMAKE_BUILD_TYPE ${CMAKE_BUILD_TYPE} CACHE STRING "Choose build type: None D
 project(volk)
 
 enable_language(CXX)
-set(CMAKE_CXX_STANDARD 11)
-
 enable_language(C)
-set(CMAKE_C_STANDARD 11)
+
+if(NOT (CMAKE_VERSION VERSION_LESS "3.1"))
```

Comment:
We should require CMake 3.8 right at the beginning of this CMakeLists.txt and thus we don't need to check for older versions anymore. Currently, this is still at 2.8.12 but I issued a PR to bump the minimum Cmake version to 3.8 which makes sense to align VOLK and GR requirements.

Reply:
We've now set the min CMake version to 3.8 ... so please fix up this clause.